src/Ads/index.js

```diff
@@ -16,223 +16,77 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import Log from '../Log'
-import { mediaUrl } from '../VideoPlayer'
-
-let consumer
-
-let getAds = () => {
-  // todo: enable some default ads during development, maybe from the settings.json
-  return Promise.resolve({
-    prerolls: [],
-    midrolls: [],
-    postrolls: [],
-  })
-}
+import AdsPlayer from './player'
+import { initAds as initAdsPlayer } from './player'
 
 export const initAds = config => {
   if (config.getAds) {
-    getAds = config.getAds
+    initAdsPlayer(config)
   }
 }
 
-const state = {
-  active: false,
+let getAdvertisingId = function() {
+  return '26ccd5a7b2c2a50e7d4b2244e9d4c048'
 }
 
-const playSlot = (slot = []) => {
-  return slot.reduce((promise, ad) => {
-    return promise.then(() => {
-      return playAd(ad)
-    })
-  }, Promise.resolve(null))
+let clearAdvertisingId = function() {
```

Comment:
I think this is better called `resetAdvertisingId` as you still have an ad id at the end of it, just a new one.

Reply:
oh, good feedback